webapp/channels/src/components/admin_console/admin_definition_ldap_wizard.tsx

```diff
@@ -342,6 +343,22 @@ export const ldapWizardAdminDefinition: LDAPAdminDefinitionConfigSchemaSettings
                     },
                 ],
             },
+            {
+                type: 'button',
+                action: ldapTestFilters,
+                key: 'LdapSettings.TestFilters',
+                label: defineMessage({id: 'admin.ldap.testFiltersTitle', defaultMessage: 'Test Filters'}),
+                help_text_markdown: false,
+                error_message: defineMessage({id: 'admin.ldap.testFiltersFailure', defaultMessage: 'We failed to apply some filters.'}),
```

Comment:
I'd probably use a more neutral tone on this one, such as `Some filters could not be applied.`

Reply:
I'll add a todo for the UX pass next week, thanks.